pkg/ccl/backupccl/backup_destination.go

```diff
@@ -136,7 +143,19 @@ func resolveDest(
 	if err != nil {
 		return "", "", "", nil, nil, err
 	}
-	if !exists {
+	if exists && collectionFullBackup {
+		// implies we're attempting to conduct a full backup into a time formatted subdirectory that
+		// already contains full backups (likely bc this backup is AS OF SYSTEM TIME)
+		return "",
+			"",
+			"",
+			nil,
+			nil,
+			errors.Newf("A full backup already exists in %s. "+
```

Comment:
Should we use `errors.Hintf` here considering we are giving the users a hint of what they can do?